src/cache/precipitation_precomputed_quantities.jl

```diff
@@ -506,6 +507,67 @@ function set_precipitation_velocities!(
     return nothing
 end
 
+function set_precipitation_velocities!(
+    Y, p, ::NonEquilMoistModel, ::Microphysics2MomentP3,
+)
+    ## liquid quantities (2M warm rain)
+    (; ᶜwₗ, ᶜwᵢ, ᶜwᵣ, ᶜwnₗ, ᶜwnᵣ, ᶜwₜqₜ, ᶜwₕhₜ, ᶜts, ᶜu) = p.precomputed
```

Comment:
ᶜwᵢ is not needed?

Reply:
Nevermind, I see it's used later. But maybe still better to grab it with all the other ice variables?